### PR TITLE
See you at re invent

### PR DESCRIPTION
**Pull request by mooreds**
Blog post about FusionAuth attending re:invent.

### CODE REVIEW

File: site/_posts/2021-11-23-fusionauth-re-invent.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+layout: blog-post
+title: "We'll see you at AWS re:Invent"
+description: FusionAuth is going to AWS re:Invent. Stop by and see us!
+author: Dan Moore
+image: blogs/fusionauth-aws-reinvent-2021/see-you-at-aws-reinvent-header-image.png
+category: blog
+excerpt_separator: "<!--more-->"
+---
+
+AWS re:Invent is one of the largest technical conferences. In the cloud space, it is the go-to event. In 2019, there were north of 50,000 attendees; this year the number of attendees is lower, but there'll still be a lot of folks in Las Vegas, NV the week after Thanksgiving. The FusionAuth team will be among them, and we'd love to see you.
```
Comment (AkiraBrand) [inline site/_posts/2021-11-23-fusionauth-re-invent.md:11]: I'd say "largest technical conferences _in the world_" or "_in the US_"

Comment (AkiraBrand) [inline site/_posts/2021-11-23-fusionauth-re-invent.md:11]:
```suggestion
AWS re:Invent is one of the largest technical conferences in the United States. In the cloud space, it is the go-to event. This year the number of attendees is rapidly approaching 20,000; there'll be a lot of folks in Las Vegas, NV the week after Thanksgiving! The FusionAuth team will be among them, and we'd love to see you.
```

File: site/_posts/2021-11-23-fusionauth-re-invent.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+We're also excited to chat with attendees. The AWS CIAM solution, [Amazon Cognito](https://aws.amazon.com/cognito/), has some issues (as [some in the FusionAuth community](/blog/2020/11/18/reconinfosec-fusionauth/) have shared). We'd like to hear from developers who are current Cognito users and are considering other solutions, such as FusionAuth. Migrating away from Cognito isn't straightforward, but there are ways.
+
+We're also looking forward to specific sessions. I personally am excited about "What's new is Amazon RDS", "Creating the best developer experience" and "Deep dive into high performance networking on AWS". While many session videos are pushed to the Amazon Web Services Youtube Channel (here's a favorite of mine from recent years: [Becoming an AWS Policy Ninja using AWS IAM and AWS Organizations](https://www.youtube.com/watch?v=hETtXCqX_Zc)), I find the in-person experience to be superior in a number of ways.
```
Comment (AkiraBrand) [inline site/_posts/2021-11-23-fusionauth-re-invent.md:19]: nice. good callout to past sessions.

File: site/_posts/2021-11-23-fusionauth-re-invent.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+Of course, it isn't all educational talks. There's also fun to be had. One activity worth attending is [Corey Quinn's nature walk](https://twitter.com/QuinnyPig/status/1461533923855331329) where he skewers Expo vendors. There are numerous parties. If you registered, chances are at least one vendor has emailed you with an invite. This twitter account, [@reInventParties](https://twitter.com/reInventParties/), is another way to discover them. Finally, someone recommended getting into nature and checking out [Red Rock Canyon](https://www.blm.gov/programs/national-conservation-lands/nevada/red-rock-canyon).
```
Comment (AkiraBrand) [inline site/_posts/2021-11-23-fusionauth-re-invent.md:21]:
```suggestion
Of course, it isn't all educational talks. There's also fun to be had! One activity worth attending is [Corey Quinn's nature walk](https://twitter.com/QuinnyPig/status/1461533923855331329) where he skewers Expo vendors. There are also numerous parties. If you registered, chances are at least one vendor has emailed you with an invite. This Twitter account, [@reInventParties](https://twitter.com/reInventParties/), is a great way to discover them. Finally, if getting into nature is more your thing, we've heard [Red Rock Canyon](https://www.blm.gov/programs/national-conservation-lands/nevada/red-rock-canyon) is a great place to check out.
```

**Review by AkiraBrand — COMMENTED** [review]
Looks great! I added a couple of suggestions.

File: site/_posts/2021-11-23-fusionauth-re-invent.md (hunk 1 above)
Comment (mooreds) [reply site/_posts/2021-11-23-fusionauth-re-invent.md:11]: Thanks, I tweaked it a bit, good feedback to not focus on the lower number.

**Pull request merged**